src/registrar/tests/models/dns/test_dns_record.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase
+from registrar.models import Domain, DnsAccount, DnsZone, DnsRecord
+
+
+class DnsZoneTest(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.dns_domain = Domain.objects.create(name="dns-test.gov")
+        self.dns_account = DnsAccount.objects.create(name="acct-base")
+        self.dns_zone = DnsZone.objects.create(dns_account=self.dns_account, domain=self.dns_domain)
+        self.dns_record = DnsRecord.objects.create(dns_zone=self.dns_zone)
+
+    def tearDown(self):
+        super().tearDown()
+        DnsAccount.objects.all().delete()
+        DnsZone.objects.all().delete()
+        Domain.objects.all().delete()
+        DnsRecord.objects.all().delete()
+
+    def test_create_multiple_dns_record_per_zone(self):
+        """Successfully create multiple DNS records for same zone."""
+        DnsRecord.objects.create(dns_zone=self.dns_zone)
```

Comment:
From the title of this test, it seems like this is testing the one-to-many relationship. Is that right? I don't think we need to check for that- we should be able to trust that the one-to-many relationship works, though I'm interested to hear your thought process.